src/components/forms/account/AccountForm.tsx

```diff
@@ -352,4 +354,5 @@ async function createBalance(data: FormValues, account: Account) {
 
   // Opening balances affect net worth
   mutate('/api/monthly-totals', undefined);
+  mutate('/api/txs/latest', undefined);
```

Comment:
latest transactions may be updated due to opening balance tx being one of the latest, so refreshing this.